browser/ui/webui/brave_rewards_page_ui.cc

```diff
@@ -1471,6 +1471,7 @@ void RewardsDOMHandler::OnStatementChanged(
 }
 
 void RewardsDOMHandler::OnAdRewardsChanged() {
+  AllowJavascript();
```

Comment:
IIUC, we shouldn't be calling `AllowJavascript` from an observer method. If JS is not currently allowed, then I think the correct behavior is to no-op here?